struct IntKey {
1:map<byte, string> byte_key;
2:map<i16, string> i16_key;
3:map<i32, string> i32_key;
4:map<i64, string> i64_key;
}
